package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@edx/edx-proctoring",
   "//": "Note that the version format is slightly different than that of the Python version when using prereleases.",
-  "version": "5.0.1",
+  "version": "5.1.0",
```

A patch version upgrade should be enough for this change.

```suggestion
  "version": "5.0.2",
```